cli/src/config/templates.toml

```diff
@@ -452,7 +452,13 @@ separate(" ",
   format_short_change_id_with_hidden_and_divergent_info(commit),
   label("author", format_user_redacted(commit.author())),
   format_timestamp(commit_timestamp(commit)),
-  commit.bookmarks().map(|bookmark| concat("bookmark-", hash(bookmark.name()).substr(0, 4))),
+  commit.bookmarks().map(|bookmark| concat(
+    "bookmark-", hash(bookmark.name()).substr(0, 4),
+    if(bookmark.remote(),
+      "@remote-" ++ hash(bookmark.remote()).substr(0, 4),
+      if(!bookmark.synced(), "*")
```

Comment:
> For getting the "*"

Since redacted template exists for debugging purpose, I think it's also good to just dump both `local_bookmarks()` and `remote_bookmarks()` (instead of omitting synced pairs and add `*` marker.)

Reply:
Since we wouldn't add `builtin_log_<style>_redacted` for each builtin `style`, I don't think the output format really matters. I'm okay with the current form, though.